pact/policy-manager/policy-manager.pact

```diff
@@ -207,7 +207,7 @@
           (quote (get-quote-info sale-id))
           (reserved (at 'reserved quote)))
           (enforce (= "" reserved) "Sale is reserved, unable to withdraw")
-          (map-withdraw token seller amount sale-id (at 'policies token))
+          (let ((_ ""))(map-withdraw token seller amount sale-id (at 'policies token)) true)
```

Comment:
What is the reason for this variable, just to return a single boolean variable right?
If so, maybe we can change it to this to make it more clear (and also don't have an unused let)?

```suggestion
          (map-withdraw token seller amount sale-id (at 'policies token))
          true
```

Reply:
This was to fix the type error, caused by `[(map-withdraw .. )]` resulting in `[[bool]]` type but there was an easy way around, by simply removing the list and return `map-withdraw` .  Pushed the changes.